example/startall.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Start mysqld
+docker-entrypoint.sh mysqld &
+sleep 1
```

Comment:
Question, is 1s enough? What are the factors for length of sleep time?

Reply:
below lines will wait until the connection is ready. So sleep 1 here is OK~

Reply:
Can we then remove line 5 as we have the until loop to wait for the DB connection.

Reply:
Some times mysqld starts very fast, so line 5 here can reduce the annoying error logs.

Reply:
I see, we could use 1s for sleep time for now, but it would be nice to explicitly catch/handle the MySQL connection error instead of waiting for 1s. Guess it is hard to decide if 1s is enough for the wait before entering into until execution. 

:-)